src/main/scala/cromwell/engine/package.scala

```diff
@@ -19,7 +19,7 @@ package object engine {
   type WorkflowId = UUID
 
   sealed trait WorkflowState {
-    def isTerminal: Boolean
+    val isTerminal: Boolean
```

Comment:
I was curious if that was intentional despite what I said, it sounds like "no" :)

I think there *are* cases where a trait should enforce val-ness and I was on the fence on if I thought this is one of them. I sided with "no" but could be easily swayed to "yes"